express/middleware/error-handler.js

```diff
@@ -0,0 +1,12 @@
+const httpStatus = require('http-status');
+const { ValidationError } = require('express-validation');
+
+module.exports = function (err, req, res, next) {
+    console.log(err.details);
```

Comment:
Is it a chance to implement Logger and move console.log there

Reply:
I didn't work with Logger before
is it a npm module?

Reply:
You could create a service for that. Just to have one place in project to interact with console. In future it'll be much simpler to justify rules for logging or not. Or switch to the third party service.